README.md

```diff
@@ -235,12 +235,17 @@ access is granted to requester. Otherwise request is aborted with an 401 or 403
   ...
 }
 ~~~
-The `jwt.clientDataClaims` option key is optional and defines in which claims the clientId and role of the user can be
-found. If this key is not specified the fields are "name" for the clientId and "role" for the role.
 The `jwt.encrpytion` option key is optional and defines which encryption was used to sign the token. If this key is not
 present, the fallback "ES256" is used. This library supports all algorithms that [lcobucci/jwt](https://github.com/lcobucci/jwt)
 version 3.* supports.
 
+The `jwt.clientDataClaims` option key is optional and defines in which claims the clientId and role of the user can be
+found. If this key is not specified the fields are "name" for the clientId and "role" for the role.
+
+The `jwt.claims` settins are used to validate the content of a token. Following constrains values from [lcobucci/jwt](https://lcobucci-jwt.readthedocs.io/en/3.4.x/validating-tokens/#available-constraints) are supported:
```

Comment:
means this 'settings' or is 'settins' correct?

Reply:
good catch I will fix the spelling mistake in my next commit

Reply:
fixed